src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/proposal/RemoveElementsProposal.java

```diff
@@ -41,10 +43,23 @@ protected RemoveElementsProposal(String label, PsiFile sourceCU, PsiFile invocat
         this.elementsToRemove = elementsToRemove;
     }
 
+    protected RemoveElementsProposal(String label, PsiFile sourceCU, PsiFile invocationNode,
+                                     PsiModifierListOwner binding, int relevance,
+                                     List<? extends PsiElement> elementsToRemove, boolean isFormatRequired) {
+        super(label, CodeActionKind.QuickFix, relevance);
+        this.sourceCU = sourceCU;
+        this.invocationNode = invocationNode;
+        this.binding = binding;
+        this.elementsToRemove = elementsToRemove;
+        this.isFormatRequired =isFormatRequired;
```

Comment:
Formatting :-) Should have consistent spacing throughout the source file.

Reply:
Modifications done. Please have a look.